From 58c2cad3cd88d2a760e631f118effd840d65197e Mon Sep 17 00:00:00 2001
From: Johan Mossberg <johan.mossberg@intel.com>
Date: Wed, 21 May 2014 01:54:13 +0200
Subject: [PATCH 041/117] REVERTME gc2235: Add board specific configuration
 data for MRD7

This should be done through EFI variables, not hard coded like in this
patch, hence the "revert me".

Issue: GMIN-944
Change-Id: I2742a288beec37e35b33abc62661bd94bf3b9517
Signed-off-by: Johan Mossberg <johan.mossberg@intel.com>
---
 arch/x86/platform/intel-mid/atomisp_gmin_platform.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
index 724beb7..10dd8dc 100644
--- a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
+++ b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
@@ -140,11 +140,21 @@ static const struct gmin_cfg_var ffrd8_vars[] = {
 	{},
 };
 
+static const struct gmin_cfg_var mrd7_vars[] = {
+	{ "INT33F8:00_CamType",  "1" },
+	{ "INT33F8:00_CsiPort",  "1" },
+	{ "INT33F8:00_CsiLanes", "2" },
+	{ "INT33F8:00_CsiFmt",   "13" },
+	{ "INT33F8:00_CsiBayer", "0" },
+	{},
+};
+
 static const struct {
 	const char *dmi_board_name;
 	const struct gmin_cfg_var *vars;
 } hard_vars[] = {
 	{ "BYT-T FFD8", ffrd8_vars },
+	{ "TABLET", mrd7_vars },
 };
 
 
-- 
1.9.3

